docs/index.md

```diff
@@ -294,101 +337,344 @@ Follow these steps:
+
+    ```bash
+    vault write astra/role role_name="R_W_User" role_id="946cfbde-24cc-4953-9355-d57bfd61bf39" \
+    org_id="ccd999999_facd_4ad3_bbb99903d999999999999999d" ttl=3600 max_ttl=36000
+    ```
+
+    *Output*:
+
+    ```bash
+    Success! Data written to: astra/role
+    ```
+
+    *NOTE*: After the role is created, you can change the defined `ttl` and `max_ttl` values by calling the same API with the same `role_name` and `org_id` arguments, and new `ttl` and/or `max_ttl` value(s).
 	
-5. Define a default lease time. Example:
+7. To read a role:
+
+    ```bash
+    vault read astra/role org_id="ccd999999_facd_4ad3_bbb99903d999999999999999d" role_name="api_r_w_user"
+    ```
+
+    *Output*:
+
+    ```bash
+    max_ttl      10h0m0s
+    org_id       ccd999999_facd_4ad3_bbb99903d999999999999999d
+    role_id      946cfbde-24cc-4953-9355-d57bfd61bf39
+    role_name    api_r_w_user
+    ttl          1h0m0s
+    ```
+
+8. To generate a token:
+
+    ```bash
+    vault write astra/org/token org_id="<ORG ID>" role_name="<ROLE NAME>" logical_name="<LOGICAL NAME>"
+    ```
+
+    *Output*:
+
+    ```bash
+    Key                Value
+    ---                -----
+    lease_id           astra/org/token/ove4ll9sshXFL3qK6I1qPezb
+    lease_duration     1h
+    lease_renewable    true
+    clientId           DTEUyKUZZrvFzzLzPGCbrbHO
+    generatedOn        2022-09-21T17:48:15.260897295Z
+    logicalName        org1
+    metadata           <nil>
+    orgId              ccd999999_facd_4ad3_bbb99903d999999999999999d
+    roleName           r_w_user
+    token              AstraCS:DTEUyKUZZrvFzzLzPGCbrbHO:4ebdef6dbbcada2f4618c9e4b6800c17b10c...
+    ```
+
+    After the lease expires, the token is automatically revoked; it's deleted from the Vault and from Astra DB.
+
+9. To read a generated token using `client_id` and `org_id`:
+
+    ```bash
+    vault read astra/org/token client_id="ZenclHmfKBenxQZjRNhRUbLc" org_id="ccd999999_facd_4ad3_bbb99903d999999999999999d"
+    ```
+    
+    *Output*:
+
+    ```bash
+    clientId       ZenclHmfKBenxQZjRNhRUbLc
+    generatedOn    2022-09-21T17:40:29.265975403Z
+    logicalName    org1
+    metadata       <nil>
+    orgId          ccd999999_facd_4ad3_bbb99903d999999999999999d
+    roleName       r_w_user
+    token          AstraCS:ZenclHmfKBenxQZjRNhRUbLc:a4bf8ca1bf164698e2631d353a295be27187e5750afc4080a4749fcc68f99f38
+    ```
+
+10. To read a token using `org_id`, `role_name`, and `logical_name`:
+
+    ```bash
+    vault read astra/org/token org_id=”ccd999999_facd_4ad3_bbb99903d999999999999999d” \
+    role_name=”r_w_user” logical_name=”org1”
+    ```
+
+    *Output*:
+
+    ```bash
+    clientId       ZenclHmfKBenxQZjRNhRUbLc
```

Comment:
Did you change any of the token or orgIDs just in case someone tries to use them?

Reply:
Yes, they're outdated or fake.